packages/about-you/theme/pages/Product.vue

```diff
@@ -0,0 +1,513 @@
+<template>
+  <div id="product">
+    <SfBreadcrumbs
```

Comment:
what exactly are the differences between this product.vue and default one? we should all we can to avoid overriding pages, especially as crucial as pdp or plp

Reply:
default one gets 500 after getting a not empty product, so, for now, I would leave it to see the product parameters and after implementing related products and other needed stuff we will delete this custom page.